build/.vsts-pr.yml

```diff
@@ -18,3 +20,38 @@ jobs:
   - template: build.yml
     parameters:
       packageArtifacts: false
+
+- job: Docker
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: DockerCompose@0
+    displayName: Container registry login
+    inputs:
+      containerregistrytype: Azure Container Registry
+      azureSubscriptionEndpoint: $(azureSubscriptionEndpoint)
+      azureContainerRegistry: $(azureContainerRegistry)
+
+  - task: DockerCompose@0
+    displayName: Build FHIR Server Image
+    inputs:
+      action: Build services
+      dockerComposeFile: samples/docker/docker-compose.yaml 
+      projectName: fhirserverforazure
+      qualifyImageNames: true
+      additionalImageTags: |
+        pr$(system.pullRequest.pullRequestNumber)
```

Comment:
Does this build image from PR branch or master?

Reply:
For now, the PR branch. When it builds off of CI it will want to pull from master.